tasks/cephfs/fuse_mount.py

```diff
@@ -68,6 +68,13 @@ def mount(self):
         run_cmd.extend(run_cmd_tail)
 
         def list_connections():
+            self.client_remote.run(
```

Comment:
Do we still even need the mount call here?  I don't know the details but I always assumed this was something that fuse was supposed to take care of for us, and if it isn't present then we can assume there are no fuse connections.

If it is necessary for some reason, then the right place to call this is probably at the start of mount() rather than in each list_connections()